tox.ini

```diff
@@ -21,80 +24,121 @@ extras =
     test
 
 commands =
-    pytest --cov=src --cov-report=xml --html=reports/pytest/report-{envname}.html --doctest-glob=*.rst --self-contained-html {posargs}
+    pytest --cov=src --cov-report=xml --html=reports/pytest/report-{envname}.html --self-contained-html {posargs}
```

Comment:
Why `--doctest-glob=*.rst` was removed? I have added it so that the `README.rst` are run, too, using pytest.

Reply:
I will re add that.